name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/WirBankPDFExtractor.java

```diff
@@ -64,8 +64,8 @@ private void addDepositTransaction()
                         // Gutschrift: Valuta 15.01.2019 CHF 2'150.00
                         // @formatter:on
                         .section("date", "amount", "currency") //
-                        .find("(Einzahlung|Deposit) 3a") //
-                        .match("^(Gutschrift: Valuta|Credit: Value date) (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<currency>[\\w]{3}) (?<amount>[\\.,'\\d]+)$") //
+                        .find("(Einzahlung|Deposit|Versement) 3a") //
+                        .match("^(Gutschrift: Valuta|Credit: Value date|Montant crédité: Valeur) (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<currency>[\\w]{3}) (?<amount>[\\.,'\\d]+)$") //
```

Comment:
```suggestion
                        .match("^(Gutschrift: Valuta|Credit: Value date|Montant cr.dit.: Valeur) (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<currency>[\\w]{3}) (?<amount>[\\.,'\\d]+)$") //
```
Special characters are to be devalued with a dot as a regular expression.